cypress/integration/stories/opa.spec.ts

```diff
@@ -126,18 +138,21 @@ describe('OPA Story', () => {
     ClustersPage.getTabCard('OPA Gatekeeper Config').should(Condition.Exist);
   });
 
-  it('should wait for initial machine deployment to be created and healthy', () => {
+  it('should wait for initial machine deployment to be created', () => {
     TrafficMonitor.newTrafficMonitor().method(RequestType.GET).url(Endpoint.MachineDeployments).interceptAndWait();
     ClustersPage.getMachineDeploymentList().should(Condition.Contain, initialMachineDeploymentName);
```

Comment:
Traffic monitor might be impacting the mocks. I'm not sure about it but I'd remove it and verify it again. It should not be required as line number 143 should wait for machine deployment.

Reply:
Okay, will try 👍

Reply:
If is doesn't work leave the comment and then we can go back to it at some point.